Pcap++/src/PcapFilter.cpp

```diff
@@ -170,17 +169,15 @@ void IPFilter::convertToIPAddressWithLen(std::string& ipAddrmodified, int& len)
 	}
 	else if (ipAddr.isIPv4())
 	{
-		const IPv4Address& ip4Addr = ipAddr.getIPv4();
-		uint32_t addrAsInt = ip4Addr.toUInt();
-		uint32_t mask = ((uint32_t)-1) >> ((sizeof(uint32_t)*8)-m_Len);
+		uint32_t addrAsInt = ipAddr.getIPv4().toUInt();
+		uint32_t mask = (uint32_t)(-1) >> ((sizeof(uint32_t) * 8) - m_Len);

```

Comment:
Very hard to understand what does this code

Reply:
yes I agree. Any suggestion on how to make that more clear?

Reply:
I have made the changes in the latest commit